src/components/icon-tile/index.tsx

```diff
@@ -16,13 +16,15 @@ function IconTile({
   children,
   size = 'medium',
   brandColor = 'neutral',
+  className,
 }: IconTileProps): React.ReactElement {
   return (
     <span
       className={classNames(
         s.root,
         s[`size-${size}`],
-        s[`color-${brandColor}`]
+        s[`color-${brandColor}`],
+        className && className
```

Comment:
🚧 `classNames` automatically handles checking if one of it's arguments is defined, so we can just pass `className` here!

```suggestion
        className
```